test-utils/src/dev/clickhouse.rs

```diff
@@ -23,6 +23,9 @@ use crate::dev::poll;
 // Timeout used when starting up ClickHouse subprocess.
 const CLICKHOUSE_TIMEOUT: Duration = Duration::from_secs(30);
 
+// Timeout used when starting a ClickHouse keeper subprocess.
+const CLICKHOUSE_KEEPER_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
Have you been observing longer times for the keeper to start up? That feels a bit odd, I generally observed that they spun up quicker than the servers.

Reply:
No I think this was during my attempts to understand the failures in CI. I can revert it.

Reply:
Done in 6b3d86622